lib/jekyll/renderer.rb

```diff
@@ -139,6 +139,11 @@ def place_in_layouts(content, payload, info)
       output = content.dup
       layout = site.layouts[document.data["layout"]]
 
+      if layout
+        layout_source = layout.path.start_with?(site.source) ? :site : :theme
+        Jekyll.logger.debug("Layout source:", layout_source)
```

Comment:
I'd much rather this be folded off into a new line without the variable being created for the GC to handle.